package.json

```diff
@@ -5,8 +5,13 @@
 	"private": true,
 	"homepage": "https://github.com/microsoft/atlas-design",
 	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1"
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"lint-css": "yarn workspace @microsoft/atlas-css lint",
+		"pretty-quick-css": "yarn workspace @microsoft/atlas-css pretty-quick"
```

Comment:
also good to have a `yarn run prettier` command which formats all files. Often need this after updating to a new version of prettier.

Reply:
added, along with prettier-check